src/PostgREST/App.hs

```diff
@@ -280,10 +280,10 @@ augumentRequestWithJoin schema allRels request =
 
 buildReadRequest :: [Relation] -> ApiRequest -> Either Text ReadRequest
 buildReadRequest allRels apiRequest  =
-  augumentRequestWithJoin schema rels =<< first formatParserError (foldr addFilter <$> (addOrder <$> readRequest <*> ord) <*> flts)
+  augumentRequestWithJoin schema rels
+  =<< first formatParserError (foldr addFilter <$> (foldr addOrder <$> readRequest <*> ords) <*> flts)
```

Comment:
I actually didn't know that this indentation is valid Haskell. Might be clearer with the second line indented like

```haskell
augumentRequestWithJoin schema rels =<<
  first formatParserError
    (foldr addFilter <$> (foldr addOrder <$> readRequest <*> ords) <*> flts)
```

I suspect that last bit might benefit from EitherT or something but I don't know enough about how that works.